entrypoint/microservice/routes/database.py

```diff
@@ -93,6 +93,6 @@ def get_documents():
             documents[i]['fulltext_cleaned'] = documents[i]['fulltext_cleaned'][:500]
         documents[i].pop('fulltext')
 
-    DB.close_db()
+    DB.disconnect()
```

Comment:
This is one of my last changes. In order to be consistent with the other services we switched to using the service `config_db.py` for accessing the database.